plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -68,3 +80,17 @@ func Register() {
 func SinceInMicroseconds(start time.Time) float64 {
 	return float64(time.Since(start).Nanoseconds() / time.Microsecond.Nanoseconds())
 }
+
+// ReportBindingRateLimiterSaturation exports status of binding rate limiter
+// whether it's full (1) or not (0).
+func ReportBindingRateLimiterSaturation(rl util.RateLimiter) {
```

Comment:
Right now, it is a binding specific. But, yes, we can change the function to be more generic. Like ReportRateLimiterSatruation(typ string, saturated bool)?